src/IIIFPresentation/API/Features/Manifest/Requests/GetManifestHierarchical.cs

```diff
@@ -0,0 +1,50 @@
+﻿using API.Converters;
+using API.Helpers;
+using API.Infrastructure.AWS;
+using MediatR;
+using Models.API.Manifest;
+using Models.Database.General;
+using Repository;
+
+namespace API.Features.Manifest.Requests;
+
+public class GetManifestHierarchical(
```

Comment:
Similar to the previous comment about the constructor values being split into separate lines